src/test/javascript/spec/component/lecture-unit/attachment-unit/attachment-unit.service.spec.ts

```diff
@@ -93,4 +93,80 @@ describe('AttachmentUnitService', () => {
         req.flush(returnedFromService);
         expect(expectedResult.body).toEqual(expected);
     });
+
+    it('should create AttachmentUnits', async () => {
```

Comment:
```suggestion
    it('should create AttachmentUnits', fakeAsync(() => {
```
Is there a reason for not using `fakeAsync`? (Same for test below) I think we had some problems with `async` for tests in the past.

Reply:
The other existing tests in this file were with async and i did not know that we might have problems with async. I changed the tests to fakeAsync.